index.html

```diff
@@ -126,134 +96,89 @@ <h2 id="background">
+      Screen Enumeration</a> and <a href=
+      "https://github.com/spark008/window-placement/blob/master/EXPLAINER.md">Window
+      Placement</a> APIs as its new incubations in 2020.
     </p>
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-    <p>
-      The following tasks are in scope for the work of the community group.
-    </p>
-    <h3 id="scope-protocols">
-      Specifications of network protocols
+    <h3 id="local-presentation-mode">
+      Local Presentation Mode
     </h3>
-    <ol>
-      <li>The specification of a set of network protocols that allows one user
-      agent to use the Presentation API to discover and present Web content on
-      another user agent, when both user agents are connected to the same Local
-      Area Network (LAN).
-      </li>
-      <li>Protocols that implement all of the interfaces, attributes, methods,
-      algorithms, and implementation requirements of the Presentation API are
-      in scope. This includes protocols that preserve the privacy and security
-      of data exchanged between the controlling and receiving user agents.
-      </li>
-      <li>Both the 2-UA and 1-UA modes of the Presentation API are in scope.
-      </li>
-      <li>Protocols that implement all of the interfaces, attributes, methods,
-      algorithms, and implementation requirements of the Remote Playback API
-      are also in scope, for the specific case of remotely playing a
-      <code>&lt;video&gt;</code> or <code>&lt;audio&gt;</code> element with a
-      <code>src</code> or <code>source</code>s media source on a remote
-      playback device. The controlling user agent should be able to send the
-      URL of the media source or stream the media data to the remote playback
-      device.
-        <p class="note">
-          In the Remote Playback API, sending the URL for remote playback is
-          referred to as <a href=
-          "https://w3c.github.io/remote-playback/#dfn-media-flinging">media
-          flinging</a>. Sending the media data itself is referred to as
-          <a href="https://w3c.github.io/remote-playback/#dfn-media-remoting">media
-          remoting</a>.
-        </p>
-      </li>
-      <li>Streaming media between the controlling and receiving user agent is
-      in scope, as this capability underlies both 1-UA mode for the
-      Presentation API and media remoting for the Remote Playback API.
+    <ul>
+      <li>A <a href=
+      "https://github.com/mfoltzgoogle/local-presentation-api/blob/gh-pages/explainer.md">
+        Local Presentation Mode</a> is a <a href=
+        "https://w3c.github.io/presentation-api/">Presentation API</a>
+        extension that removes the restriction on presentations to use a
+        separate storage area and allows the controlling browsing context and
+        the receiving browsing context have access to each other's document
+        context similarly to the window.open API. Use cases include e.g.
```

Comment:
I would prefer to remove the phrase "similarly to the window.open API" since we may go in the direction of allowing the presentation to be hosted in the same document as the controller.

Reply:
Makes sense, removed this phrase.